go/vt/proto/vschema/vschema.pb.go

```diff
@@ -156,7 +156,8 @@ type Keyspace struct {
 	Vindexes map[string]*Vindex `protobuf:"bytes,2,rep,name=vindexes,proto3" json:"vindexes,omitempty" protobuf_key:"bytes,1,opt,name=key,proto3" protobuf_val:"bytes,2,opt,name=value,proto3"`
 	Tables   map[string]*Table  `protobuf:"bytes,3,rep,name=tables,proto3" json:"tables,omitempty" protobuf_key:"bytes,1,opt,name=key,proto3" protobuf_val:"bytes,2,opt,name=value,proto3"`
 	// If require_explicit_routing is true, vindexes and tables are not added to global routing
-	RequireExplicitRouting bool `protobuf:"varint,4,opt,name=require_explicit_routing,json=requireExplicitRouting,proto3" json:"require_explicit_routing,omitempty"`
+	RequireExplicitRouting bool              `protobuf:"varint,4,opt,name=require_explicit_routing,json=requireExplicitRouting,proto3" json:"require_explicit_routing,omitempty"`
+	Views                  map[string]string `protobuf:"bytes,5,rep,name=views,proto3" json:"views,omitempty" protobuf_key:"bytes,1,opt,name=key,proto3" protobuf_val:"bytes,2,opt,name=value,proto3"`
```

Comment:
We won't need this then, right?

Reply:
this would still be needed for internal storage, in the next PR https://github.com/vitessio/vitess/pull/11896 
I will see if this can be avoided.

Reply:
The reason I kept it is to show the e2e test and plan test works. But, may be not important.